Done Lab8

alekseevart3m opened the pull request.

alekseevart3m committed:
fixlab8_task11

alekseevart3m committed:
Add files via upload

NemirovichMark commented:
Здравствуйте, Артем.

2й номер: нужно не весь текст развернуть, а каждое слово отдельно. При этом знаки препинания оставить на своих местах.
4й номер принял.

Остальных нет. Жду.

На доработку.

alekseevart3m committed:
Fixlab8

Здравствуйте, проверти пожалуйста на ошибки , думаю что то может быть не так , но вроде все работает как нужно
